packages/react-provider/src/stories/FluentProviderFrame.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Button } from '@fluentui/react-button'; // codesandbox-dependency: @fluentui/react-button ^9.0.0-beta
+import { makeStyles } from '@fluentui/react-make-styles';
+import { FluentProvider } from '@fluentui/react-provider'; // codesandbox-dependency: @fluentui/react-provider ^9.0.0-beta
+import { teamsDarkTheme, webLightTheme } from '@fluentui/react-theme'; // codesandbox-dependency: @fluentui/react-theme ^9.0.0-beta
+
+const useStyles = makeStyles({
+  example: {
+    margin: '5px',
+  },
+  darkThemeBackground: theme => ({
+    backgroundColor: theme.colorBrandBackground,
+  }),
+});
+
+const PortalFrame: React.FunctionComponent<{
+  children: (externalDocument: Document) => React.ReactElement;
+}> = ({ children }) => {
+  const [frameRef, setFrameRef] = React.useState<HTMLIFrameElement | null>(null);
+
+  return (
+    <>
+      <iframe
+        ref={setFrameRef}
```

Comment:
not familiar with how iframe references are set. you just pass in the setter and the iframe takes care of it?

Reply:
I haven't used iframes much.  I would think it sets the ref like any other element.  I think the code is trying to only call CreatePortal once the iframe is up and references. This is ported code from the make-styles basic tests.